mmv1/third_party/terraform/services/compute/resource_compute_instance.go.erb

```diff
@@ -244,6 +246,27 @@ func ResourceComputeInstance() *schema.Resource {
 										ForceNew:     true,
 										Description:  `A map of resource manager tags. Resource manager tag keys and values have the same definition as resource manager tags. Keys must be in the format tagKeys/{tag_key_id}, and values are in the format tagValues/456. The field is ignored (both PUT & PATCH) when empty.`,
 									},
+									
+									"provisioned_iops": {
+										Type:         schema.TypeInt,
+										Optional:     true,
+										AtLeastOneOf: initializeParamsKeys,
+										Computed:     true,
+										ForceNew:     true,
```

Comment:
The provided description suggests that it is possible to update these values without recreating, but this implementation will *always* recreate these values. Will the API return an error if an update is attempted too often?

Reply:
Updated the documentation. The iops and throughput can only be updated via gcloud/api.
This path is create-on-create where the disk is created with instances. This does not support update.

On another note: For terraform disk resource, already support patch resource https://github.com/GoogleCloudPlatform/magic-modules/blob/49252d7a95dc28f660ad9be07c260f02e23d51fb/mmv1/products/compute/Disk.yaml#L445-L462